tensorboard/webapp/hparams/_redux/types.ts

```diff
@@ -43,6 +45,8 @@ export const HPARAMS_FEATURE_KEY = 'hparams';
 
 export interface HparamsState {
   specs: ExperimentToHparams;
+  currentSpecs: HparamAndMetricSpec;
```

Comment:
Could you elaborate a bit on what the final data model is going to look like? Introduction of "currentSpecs" and "sessionGroups" isn't clearly going in the direction that I hoped this would. It could just be a naming issue, though.

Ultimately I think the data model should look something like:
```
export interface HparamsState {
  listSpecs: Record<string, HparamAndMetricSpec> ;
  listSessionGroups: Record<string, SessionGroup[]>;
  listFilters: Record<string, Filters>;
  dashboardSpecs: HparamAndMetricSpec;
  dashboardSessionGroups: SessionGroup[];
  dashboardFilters: Filters;
}
```

Notably, there is no notion of a single "current" spec since (1) there is also no notion of a "previous" spec and (2) you could have multiple active lists at the same time - multiple in the experiment list and one in the dashboard.

Reply:
I do think this is a naming issue. I agree that `dashboardSpecs` seems like a better name as it helps clarify the usease. I chose `ccurrentSpecs` to indicate that it's value would change as the experiment ids changed, but in hindsight that is implicit with redux properties.